src/hotspot/share/runtime/continuation.cpp

```diff
@@ -311,6 +311,9 @@ void Continuation::set_cont_fastpath_thread_state(JavaThread* thread) {
   assert(thread != nullptr, "");
   bool fast = !thread->is_interp_only_mode();
   thread->set_cont_fastpath_thread_state(fast);
+  if (thread->is_interp_only_mode() && ContinuationEntry::enter_special() != nullptr) {
+    ContinuationEntry::enter_special()->clear_continuation_enter_special_inline_caches();
```

Comment:
Will this call impact all JavaThread's, not only the one passed in the argument?
Just want to understand this better.
Would it be worth to add a comment explaining this aspect (if applicable)?